.github/scripts/dapr_tests_summary.js

```diff
@@ -0,0 +1,11 @@
+
+module.exports = async ({ glob, core }) => {
+    const globber = await glob.create(process.env["TEST_OUTPUT_FILE_PREFIX"] + "_summary_table_*.json")
+    for await (const file of globber.globGenerator()) {
+        const testSummary = require(file) // the file is a json file so we can import
```

Comment:
Would prefer to see `await fs.readFile(file, 'utf8')` used here because using `require` to import JSON has some caveats (like with caching and possibly some other odd behaviors)

Reply:
alright, that's why I prefer [deno](https://deno.land/) xD